src/styles/index.scss

```diff
@@ -1,3 +1,121 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+p {
   margin: 0;
 }
+
+$title-color: #060b35;
+$text-color: #616070;
+$background-color: #00acdc;
+
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 auto 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $title-color;
+  }
+
+  &__text {
+    margin: 0 0 16px;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
Default value, so you can just remove it everywhere
```suggestion
```

Reply:
<img width="628" alt="image" src="https://github.com/user-attachments/assets/873543ef-6fba-490d-8880-6239c293e966">
still not fixed